src/styles/partials/components/_card-container.scss

```diff
@@ -0,0 +1,26 @@
+.dg_card {
+  border: $border-width-small solid $gray;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: space-between;
+  position: relative;
+  margin-bottom: $margin-large;
+  padding: $padding-large;
+  width: 100%;
+
+  .dg_icon-container {
+    color: $gold;
+    font-size: 60px;
```

Comment:
i suspect maybe this should be an icon variable?

Reply:
$largest-icon-size seemed a bit large and $large-icon-size seemed a bit small. If choosing between the two, I'd probably go with $largest-icon-size. The change has been committed to preview.